v1/content.yaml

```diff
@@ -487,8 +477,30 @@ paths:
       responses:
         '200':
           description: |
-            The html for the given page, revision and tid. Conforms to
-            https://www.mediawiki.org/wiki/Parsoid/MediaWiki_DOM_spec.
+            The html for the given page, revision and tid.
+
+            See [the MediaWiki DOM
+            spec](https://www.mediawiki.org/wiki/Parsoid/MediaWiki_DOM_spec) for a
+            description of the MediaWiki-specific semantic markup in this HTML.
+            Note that additional metadata is available in the HTML head.
+
+            This HTML can be edited using arbitrary HTML tools. The modified HTML
+            can be converted back to wikitext using the
+            [/transform/html/to/wikitext{/title}{/revision}](#!/Transforms/transform_html_to_wikitext__title___revision__post)
+            entry point.
+          schema:
+            type: string
```

Comment:
http://spacetelescope.github.io/understanding-json-schema/reference/combining.html suggests that should work, too. Patch coming.